dbt_copilot_helper/templates/pipelines/codebase/overrides/stack.ts

```diff
@@ -380,6 +384,24 @@ export class TransformedStack extends cdk.Stack {
         });
     }
 
+    private allowPipelineToUseEnvManagerRole() {
+        const pipelineRolePolicy = this.template.getResource("PipelineRolePolicy") as cdk.aws_iam.CfnPolicy;
+        (pipelineRolePolicy.policyDocument.Statement as Array<any>).push({
+            Effect: 'Allow',
+            Action: ['sts:AssumeRole'],
+            Resource: [this.envManagerRole],
+        });
```

Comment:
The policy document is the same here as in the next function, so could be pulled out.